### PR TITLE
FOPTS-911 Cost Anomaly Alerts - Billing Center as dimension

### PR DESCRIPTION
**Pull request by vadgun**
### Issues Resolved

[FOPTS-911](https://flexera.atlassian.net/browse/FOPTS-911)

### Link to Example Applied Policy

[Applied Policy](https://app.flexeratest.com/orgs/1105/automation/applied-policies/projects/107982?policyId=6491f4ca44e32e000102c701)

### Contribution Check List

- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/cloud_cost_anomaly_alerts/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.5
+
+- The Billing Center dimension was added to show the corresponding results
```
Comment (juanestrahcl) [inline cost/cloud_cost_anomaly_alerts/CHANGELOG.md:5]: This one can be more similar to other updates in the file, you could put something like:
`added billing center dimension to show the corresponding results`.

File: cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt [hunk 2]
```diff
@@ -90,7 +90,7 @@ credentials "auth_flexera" do
 end
 
 ###############################################################################
-# Datasources
+# Datasources & Scripts
```
Comment (juanestrahcl) [inline cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:93]: I think this should be only "Datasources" in the scripts section you can put "Scripts".

File: cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt [hunk 3]
```diff
@@ -234,6 +202,44 @@ script "js_filter_dimensions", type: "javascript" do
...
+      field "allocation_table", jmes_path(col_item,"allocation_table")
+    end
+  end
+end
+
+datasource "ds_top_level_billing_centers" do
+  run_script $js_top_level_bc, $ds_listed_billing_centers
+end
+
+script "js_top_level_bc", type: "javascript" do
```
Comment (juanestrahcl) [inline cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:231]: I think all scripts should be after all data sources.

**Review by juanestrahcl — COMMENTED** [review]
Aside of comments, probably you can check also "New functionality includes testing." box since you tested it with an applied policy.

File: cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt (hunk 2 above)
Comment (vadgun) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:93]: Hi Juan regarding this article; Datasources and Scripts now are presented in the order of execution followed by any script blocks that they call.
https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices

File: cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt (hunk 3 above)
Comment (vadgun) [reply cost/cloud_cost_anomaly_alerts/cloud_cost_anomaly_alerts.pt:231]: Hi Juan regarding this article; Datasources and Scripts now are presented in the order of execution followed by any script blocks that they call.
https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices

File: cost/cloud_cost_anomaly_alerts/CHANGELOG.md (hunk 1 above)
Comment (vadgun) [reply cost/cloud_cost_anomaly_alerts/CHANGELOG.md]: Comment Addressed. Thank you

**Review by XOmniverse — CHANGES_REQUESTED** [review]
There is a merge conflict that needs to be fixed before this can be merged

**Pull request closed without merging**